ml-algorithms/src/main/java/org/opensearch/ml/engine/ModelHelper.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.engine;
+
+import lombok.extern.log4j.Log4j2;
+import static org.opensearch.ml.engine.MLEngine.getLoadModelPath;
+import static org.opensearch.ml.engine.MLEngine.getModelCachePath;
+import static org.opensearch.ml.engine.MLEngine.getUploadModelPath;
+import static org.opensearch.ml.engine.utils.FileUtils.deleteFileQuietly;
+
+/**
+ * A helper class which provides common function for models.
+ */
+@Log4j2
+public class ModelHelper {
+    public static final String PT = ".pt";
```

Comment:
may be: PYTORCH_FILE_EXTENSION and ONNX_FILE_EXTENSION

Reply:
Sure, will address this in next PR